docs/diagrams/webhooks-flow.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2023-02-21T15:26:10.994Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15" etag="t5CxjTPOKw2sX4t2elmX" version="20.8.21" type="device"><diagram name="Page-1" id="j61EQ_yQ_Mo7Y-l2gbru">7Vxbd9o4EP41PJJjWbaxH4Ek7dnTdtNmL23fHCxAW2NRW05gf/1KWALLEpcEY5uz7UNqDfJtLp++GQ304HixepeGy/lHEqG4Z1vRqgdve7YNAstn/3HJWkisgVdIZimOhGwneMT/IjlRSHMcoUyZSAmJKV6qwglJEjShiixMU/KiTpuSWL3rMpwhTfA4CWNd+jeO6LyQ+vZgJ3+P8Gwu7wy8oPhkEcrJ4k2yeRiRl5II3vXgOCWEFkeL1RjFXHtSL8V593s+3T5YihJ6ygl98H3dH+X3/seH0TgfAjf5uejb4tnoWr4witj7iyFJ6ZzMSBLGdzvpKCV5EiF+VYuNdnM+ELJkQsCE/yBK18KYYU4JE83pIhafohWmX/npN64YfVNGtytx7c1gXRo8oBQvEEWplCU0XX8tD0pX4sPdpTYjeS1dd0KdGcnTCTqgMOmDYTpD9MA8WMzj2izdQFjmHSLsLdI1m5CiOKT4WfW2UDjtbDtve+oDweyRbUtEGJRxIuNLepu8RPGg4qydd7CD0mPsRBufeYX/OMXdnsM4F6/wBU0Qe5+MP0OyeZAMZdkCbZ46RT9zlFGj030Inxh4KI4SxniWsOMJO5vbfPSMUopZcA7FBwscRYVPogz/Gz5trscNvOQvvFGBO+q5t0aTHwoIfiO06hkQRtxECWLFouIs68aGDlSNU4zONHnftkxXlRcg02mGLmJroNl6HCYRjkKKNIMysFvyw3wRDyeUlG23sfMDyTDFhNvwiVBKFgbjUg4nZXcgOY1xgsZbqD8YypoJ95oKOKpGAzF82QE+FKJ5Ces9a7/pFN2/VtFem6AMXgPJjcMvPBF+3brh9yyD+lrkfOMA2VkQ9OoCwT5DQSW0JL6+FQMrl2kA9FzNdp9Id0gTODE+dyH5TYnItuITeM3wI8dSod12muVHUPOeWzThixjnR3SOSpwI3mtulc7J4ilnTzp6mWOKHpfhRvkvLNdSnaOGRRB6jqKpgastgsAyrILBpVZBPfCWKcvMUoSSLe8Qyox0zW3DDRzX3hTH8ZjEnKWwc2HkIj9ymDyjKfmBSp/49hP0vHr07bpApXEWD9KqyrfZY1nlMLiQyuXNTliofrGREoodh7ugLT7y/X0fQ5fc/ZZnP54/Dz///BM8Sz7S/QXMqLVzbFX/AgPgje8OgGMXfwMlqB1wseXGaNjgVLLSCM88mT+qFt3vsod4JrTAQKWa8Dyq2QC39DRzPTIHoZwbPKIs2+TH3WIF27psE7TA6AgdqlheknzXg3nBieuT7bS5HOkkuaB5/DweC1Oc4GxuYHlXiWT2MSQDnlMJus4Dmb7uTMhiGSbrP1hmI6BsA23XSdUddVW3ods+VXf/Hzj4FtiznVNhr3ZabiaJnqs6kOc2W4Wwr60AKb27ngoklLvQNRUhLw+ots4M2606djsPlwF/6TgeeJX0LghugvK/ZqNa33x9SAnfbeMWbzdv8EBFU56eN/iG9dK/VDVRNmwcpCj3+4hm9zmK5pp+B8qJOi3clnC/oCyP6V8oxVO2lPCt4AeURDiZXaHuq/zQMyTJzWoe2vs1f7da4vQqfRxaTtf03OpW/fn9U29fcuGpO32wW1vxUC82jEnEeOuMWYDJh3mEeb9K1axX3dDiWy13tMBBd+Kka9mqDJDjkdRQtgqhdWM5A0bh5F/FmYIqR7v0FrqevBa0oc2C4NEMVnp8PRms7amdhLIXr7sZrESgX+nqGRHv2M1EvBtU1gub5bVlCGg04B290fQToUUsdT3spdvXVLiSnlJT2Fd2Ri+PAlBv4hElik2mJyxa5H9tlyy2tQfZAOW33QEFT8yfrzKZs13VtwND1zVwDequdvDVpm6JtvvVzWCIaXyKr1PjXqUX0jelz6a0oA6Nz7+MnsajT/3599Ua/m4lzsjpt7uX3xmOYNwlNnzPyKjChirfbsV3AOClbweAgecMLNf3K0vKHoqgXRcMtO8vVZyt0FQdZOOQ/g6WhccpCq9057r69TDgG1DWVIivI+YPOXZJ4UPu55tvhVmZ6HoSjR+To4q3TqUGBkKmaXb/WgXcihoHBjWaKo91VFSMauxQ4bFLbVBGXZ3aDlBUzdvqgtI3qFrdo6+z5WlP622peVMJL0ELOlHcOORQJVPJTZXzc5izgAoO1FqdY6B4l9p2NWqqa9+eehvp6xJy7Z+31yusG9lP87qQGqZpuC5NEFCxlyRuN/ml97mVXyQ4Mh8GR+YD59B8dlA8cY14wIa7X2gopu9+6ALe/Qc=</diagram></mxfile>
```

Comment:
Is this file added intentionally?

Reply:
Same question about the pyc file, although I can't leave a comment on it directly since it's a binary file 😄

Reply:
This file is the diagram, so whenever another dev has to edit anything in the future they won't need to create it from scratch like I did. Should I remove it?

Reply:
Ah, sorry that I repeated Albina's comment from her review. I agree with her that it might be helpful to mention this in the README to clarify what it's for. But yeah, that justification is fine with me 👍

Reply:
...which, you did! So, we're all good 😄